src-docs/src/views/drag_and_drop/drag_and_drop_disable_blocking.js

```diff
@@ -30,22 +34,35 @@ export default () => {
   return (
     <EuiDragDropContext onDragEnd={onDragEnd}>
       <EuiDroppable
-        droppableId="DROPPABLE_AREA"
+        droppableId="CUSTOM_HANDLE_DROPPABLE_AREA"
         spacing="m"
         withPanel
-        grow={false}
       >
         {list.map(({ content, id }, idx) => (
           <EuiDraggable
             spacing="m"
             key={id}
             index={idx}
             draggableId={id}
-            disableInteractiveElementBlocking
+            customDragHandle={true}
+            hasInteractiveChildren={true}
           >
-            <EuiButton fullWidth onClick={() => {}}>
-              {content}
-            </EuiButton>
+            {(provided) => (
+              <EuiPanel className="custom" paddingSize="s">
+                <EuiFlexGroup alignItems="center" gutterSize="m">
+                  <EuiFlexItem grow={false}>
+                    <div {...provided.dragHandleProps} aria-label="Drag Handle">
+                      <EuiIcon type="grab" />
+                    </div>
```

Comment:
I would suggest using a transparent `EuiPanel` with a padding instead of a div.

- This would give us a bigger size draggable area.
- It would ensure more consistent spacing between the icon and the button.

```suggestion
                <EuiFlexGroup alignItems="center" gutterSize="s">
                  <EuiFlexItem grow={false}>
                    <EuiPanel
                      color="transparent"
                      paddingSize="s"
                      {...provided.dragHandleProps}
                      aria-label="Drag Handle"
                    >
                      <EuiIcon type="grab" />
                    </EuiPanel>
```

<img width="1112" alt="Frame 16@2x" src="https://user-images.githubusercontent.com/2750668/151602347-081e8e75-b2f3-478e-9944-74938a6d485e.png">

Reply:
Thank you @miukimiu! These suggestions were easy to implement, and made the button hit space larger and more consistent. Ready for next review.